fox_scraper/fox_scraper.py

```diff
@@ -48,63 +72,54 @@ def __init__(self, site_address):
     def get_web_content(self, http_address):
```

Comment:
yes and no...
it function get the web content and returns a parser way to read this content...
so... maybe...